hivemq-edge/src/frontend/src/modules/Workspace/components/edges/ObservabilityEdgeCTA.spec.cy.tsx

```diff
@@ -0,0 +1,24 @@
+import ObservabilityEdgeCTA from './ObservabilityEdgeCTA.tsx'
+
+describe('ObservabilityEdgeCTA', () => {
+  beforeEach(() => {
+    cy.viewport(400, 400)
+  })
+
+  it.only('should render the button', () => {
+    cy.mountWithProviders(<ObservabilityEdgeCTA source="1" onClick={cy.stub().as('onClick')} />)
+    cy.get('button').eq(0).should('not.have.attr', 'aria-describedby')
+    cy.get('button').eq(0).focus()
+    cy.get('button').eq(0).should('have.attr', 'aria-describedby')
```

Comment:
If possible, we should test the value of the attribute in this case, wdyt?
something like:
```
cy.get(`[aria-describedby="the id of the descriptor"]`)
```